### PR TITLE
[3.x] Improve user communications in the theme editor

### PR DESCRIPTION
**Pull request by YuriSizov**
A `3.x` version of https://github.com/godotengine/godot/pull/51242. Doesn't include the type variations change because we don't have them in 3.x, obviously.

### CODE REVIEW

File: editor/plugins/theme_editor_plugin.cpp [hunk 1]
```diff
@@ -2888,7 +2925,7 @@ void ThemeEditor::_theme_save_button_cbk(bool p_save_as) {
 }
 
 void ThemeEditor::_theme_edit_button_cbk() {
-	theme_edit_dialog->popup_centered(Size2(850, 760) * EDSCALE);
+	theme_edit_dialog->popup_centered(Size2(850, 700) * EDSCALE);
```
Comment (Calinou) [inline editor/plugins/theme_editor_plugin.cpp:2928]: Does a 700 pixel-tall popup still fit on a 1366×768 monitor with the Godot editor maximized, but not fullscreen? Assuming it's run on Windows with a non-autohiding task bar and a visible window border, that is.

**Review by Calinou — APPROVED** [review]
Looks good to me, I just have one question about the popup size. Perhaps `popup_centered_minsize()` could be used instead, or the popup could be made resizable in a future PR.

File: editor/plugins/theme_editor_plugin.cpp (hunk 1 above)
Comment (pycbouh) [reply editor/plugins/theme_editor_plugin.cpp:2928]: It should, I think. The user who reported it not fitting shared a screenshot where it was almost okay (but they also had a weird combination of Windows and Godot Editor scaling, so I'm not entirely sure how representative it is).

**Comment by pycbouh** [comment]
I don't have anything against making it resizable, but it's still a fairly large popup, so just making it resizable wouldn't solve the problem. I think it should be good now, and if indeed someone wants it to be resizable it can be done in a separate PR, I think.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**